libseccomp/src/lib.rs

```diff
@@ -602,6 +602,15 @@ impl ScmpFilterContext {
         Ok(attribute)
     }
 
+    /// get_no_new_privs_bit returns the current state the No New Privileges bit will be set
+    /// to on the filter being loaded, or an error if an issue was encountered
+    /// retrieving the value.
+    pub fn get_no_new_privs_bit(&self) -> Result<bool> {
```

Comment:
```suggestion
    pub fn no_new_privs_bit(&self) -> Result<bool> {
```

See https://rust-lang.github.io/api-guidelines/naming.html#c-getter.

---

I would also move this function between `set_filter_attr` and `set_no_new_privs_bit` to keep the get and the set method together.

Reply:
@rusty-snake  Thank you for the review. Regarding the naming of the function name, I'd like to conform to [libseccomp-golang](https://github.com/seccomp/libseccomp-golang) as follows.
https://github.com/seccomp/libseccomp-golang/blob/main/seccomp.go#L845.
What do you think?

>I would also move this function between set_filter_attr and set_no_new_privs_bit to keep the get and the set method together.
I'll change the order. Thanks!

Reply:
I'm not sure. Trying to have a very similar libseccomp API across different languages seems to wrong. But following the Rust API Guidelines to create an API desgin which Rustaceans except/are accustomed with isn't wrong either.

Reply:
Yes, but I think `get_no_new_privs_bit` is a little different from `getter` because `no_new_privs_bit` is not managed as filed value in `ScmpFilterContext`. Of course, I have to change test name (`test_filter_attribute_getters_and_setters`) that is just taken from libseccomp-golang.
I agree with changing all getter and setter functions name in case of `notify.fs` https://github.com/libseccomp-rs/libseccomp-rs/blob/main/libseccomp/src/notify.rs#L67